src/modules/inspect/inspect-correlation/components/diagram-viewer/diagram-viewer.ts

```diff
@@ -119,7 +120,7 @@ export class DiagramViewer {
     await this._importXml(this._diagramViewer, this._uncoloredXml);
   }
 
-  public processModelIdChanged(): void {
+  public activeDiagramChanged(): void {
     if (this._diagramViewer === undefined) {
```

Comment:
Extract Business Rule, when already touching this file you could do this as well.

```suggestion
    const diagramViewerNotSet: boolean = this._diagramViewer === undefined;
    if (diagramViewerNotSet) {
```